### ISSUE TITLE
Use cifar10 datasets get score 5.63 ?

### ISSUE THREAD

**Issue by odie2630463** [sbarratt/inception-score-pytorch#3]
Hi , i try cifar10 datasets get score 5.63 , but other code get about 12 
Is there some thing wrong ? thanks !

**Comment by sbarratt** [comment]
See http://arxiv.org/abs/1801.01973

**Comment by simopal6** [comment]
@sbarratt I read the paper but couldn't find an explanation to @odie2630463's question. Indeed, using your code I obtained an Inception score of 6.19 on CIFAR-10, but even you report results between 9 and 11...

**Comment by sbarratt** [comment]
See https://github.com/sbarratt/inception-score-pytorch/commit/3bcf441c7a3229985e3e18c7c03ea3984ff74836, which sets `transform_input=False`. That is required to get the same results.

**Comment by bknyaz** [comment]
My score for CIFAR-10 is 9.86.
Using normalization values as suggested in [pytorch docs](http://pytorch.org/docs/master/torchvision/models.html) helped me to increase the score up to 10.56, but is still below the tensorflow results, which must be around 11.24 according to https://arxiv.org/pdf/1606.03498.pdf. I think the reason for this gap is that the model in pytorch makes slightly different predictions for the same images.
